VersionControlUI/src/nodes/DiffFrame.h

```diff
@@ -66,21 +66,30 @@ class VERSIONCONTROLUI_API DiffComparisonPair : public Super<Visualization::UINo
 		Model::Text* singleObjectPath();
 
 		bool twoObjectPathsDefined();
-		bool isDummyDiffComparisonPair();
+		bool isDummyDiffFrame();
 		QString comparisonName();
 
 		QList<ObjectPathCrumbData> objectPathCrumbsDataOldNode();
 		QList<ObjectPathCrumbData> objectPathCrumbsDataNewNode();
 
 		static QString computeObjectPath(Model::Node* node);
 
+		void addOldChangedNode(Model::Node* oldNode);
```

Comment:
In the future, please do not sneak in changes that do not correspond to the description of the commit/PR.